### PR TITLE
fix(grid-core): - Generate .d.ts files for grid and scheduler core packages

### PR DESCRIPTION
**Pull request by ivanblinov2k17**

### CODE REVIEW

File: packages/dx-scheduler-core/src/utils.ts [hunk 1]
```diff
@@ -89,14 +89,17 @@ export const getAppointmentStyle: PureComputed<
...
-}) => ({
-  height,
-  width: `${width}%`,
-  transform: `translateY(${top}px)`,
-  msTransform: `translateY(${top}px)`,
-  left: `${left}%`,
-  position: 'absolute',
-});
+}) => {
+  const transform = `translateY(${top}px)` as any;
```
Comment (timbset) [inline packages/dx-scheduler-core/src/utils.ts:93]: Why do you use `any` here? Could you replace it with the `string` type?

File: packages/dx-scheduler-core/tsconfig.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "./dist",
     "downlevelIteration": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```
Comment (timbset) [inline packages/dx-scheduler-core/tsconfig.json:6]: If you succeed in avoiding `any` in a code, please, rollback the change

File: packages/dx-react-grid/api/dx-react-grid.api.md [hunk 3]
```diff
@@ -1558,9 +1558,9 @@ export class TableSummaryRowBase extends React_2.PureComponent<TableSummaryRowPr
     // (undocumented)
     static GROUP_ROW_TYPE: symbol;
     // (undocumented)
-    render(): JSX.Element;
+    render(): React_2.JSX.Element;
     // (undocumented)
-    renderContent(column: any, columnSummaries: any): JSX.Element;
+    renderContent(column: any, columnSummaries: any): React_2.JSX.Element;
```
Comment (timbset) [inline packages/dx-react-grid/api/dx-react-grid.api.md:1563]: What happened here? Why docs have changed?

**Review by timbset — CHANGES_REQUESTED** [review]

File: packages/dx-scheduler-core/src/utils.ts (hunk 1 above)
Comment (ivanblinov2k17) [reply packages/dx-scheduler-core/src/utils.ts:93]: No i can't
image

File: packages/dx-scheduler-core/tsconfig.json (hunk 2 above)
Comment (ivanblinov2k17) [reply packages/dx-scheduler-core/tsconfig.json:6]: I haven't succeed in this. It involves hundreds of types, and i don't see the point in creating them for this task

File: packages/dx-react-grid/api/dx-react-grid.api.md (hunk 3 above)
Comment (ivanblinov2k17) [reply packages/dx-react-grid/api/dx-react-grid.api.md:1563]: Probably newer version of api lib fixed this. In dx-react-grid.api file there are many occurencies of React_2 prefix so it shouldn't affect the documentation page

**Review by timbset — APPROVED** [review]

**Review by VasilyStrelyaev — APPROVED** [review]

**Pull request merged**